### PR TITLE
Group variables into a common class

### PR DESCRIPTION
**Pull request by klebertarcisio**
Hi,

I saw that both ServiceConfig and ReferenceConfig use similar variables. I believe that a good solution is to move these variables to a common class.

### CODE REVIEW

**Comment by chickenlj** [comment]
LGTM.

**Review by AlbumenJ — APPROVED** [review]
LGTM.

File: dubbo-common/src/main/java/org/apache/dubbo/config/ServiceConfigBase.java [hunk 1]
```diff
@@ -403,15 +390,7 @@ public String getUniqueServiceName() {
         return URL.buildKey(interfaceName, getGroup(), getVersion());
     }
 
-    @Override
-    public String getGroup() {
-        return StringUtils.isEmpty(this.group) ? (provider != null ? provider.getGroup() : this.group) : this.group;
-    }
 
-    @Override
```
Comment (wuwen5) [inline dubbo-common/src/main/java/org/apache/dubbo/config/ServiceConfigBase.java:411]: https://github.com/apache/dubbo/issues/7529

**Comment by wuwen5** [comment]
#7529\ #7575

**Pull request merged**